microsoftsccm_connector.py

```diff
@@ -80,28 +81,34 @@ def _get_fips_enabled(self):
             self.debug_print('FIPS is not enabled')
         return fips_enabled
 
-    def _execute_ps_command(self, action_result, ps_command):
-        """ This function is used to execute power shell command.
-
-        :param action_result: object of ActionResult
-        :param ps_command: power shell command
-        :return: output of executed power shell command
-        """
-
-        resp_output = None
+    def _get_protocol(self):
         server_cert_validation = 'ignore'
         transport = 'ntlm'
 
-        if self._get_fips_enabled():
+        if self._auth_type != MSSCCM_DEFAULT_AUTH_METHOD:
```

Comment:
So `default` is really `ntlm`?

Reply:
Default is NTLM when FIPS is enabled, otherwis it is Basic HTTP